### DISCUSSION TITLE
Allow us to disable Spark integration.

### DISCUSSION THREAD

**Discussion by ItzSave** [PaperMC/Paper#11327] [Ideas]
### Is your feature request related to a problem?

I want to be able to update spark as needed instead of relying on a paper update.

### Describe the solution you'd like.

A way to disable the spark integration and use a standard spark jar file.

### Describe alternatives you've considered.

None because paper ignores the spark file in /plugins/

**Comment by ItzSave** [comment]
An alternative I just thought of would be allowing us to define what spark version it downloads on startup.

**Comment by lynxplay** [comment]
Have a look at https://docs.papermc.io/paper/reference/global-configuration#spark

**Reply by ItzSave** [reply]
I modified those settings but it still ignores the file placed in /plugins/

**Reply by kennytv** [reply]
There's a system property you need for overriding: https://docs.papermc.io/paper/profiling#spark

**Reply by lynxplay** [reply]
Maybe worth adding that to the docs yaml part.

**Reply by ItzSave** [reply]
> There's a system property you need for overriding: https://docs.papermc.io/paper/profiling#spark

This is good to know but I do agree needs to be easier to find and know. I figured the spark section was just explaining some basics on how it functions.
